drivers/tee/optee/Kconfig

```diff
@@ -12,3 +12,11 @@ config OPTEE_BENCHMARK
 	help
 	  This enables benchmarking feature in the OP-TEE Trusted
 	  Execution Environment (TEE) driver.
+
+config OPTEE_SHM_NUM_PRIV_PAGES
+	int "Private Shared Memory Pages"
+	default 1
+	depends on OPTEE
+	help
+	  This sets the number of private shared memory pages to be
+	  used by OP-TEE Trusted Execute (TEE) driver.
```

Comment:
"by the OP-TEE driver".
What are private memory pages used for? How should the value be selected? When does it need to be increased? By how much? I think all that has to be documented here, no?

Reply:
I have written following description for --help-- section.
Please check if it is right or something needs to be corrected.

Thanks.

> This sets the number of private shared memory pages to be
 used by OP-TEE TEE driver.
> 
> This shared memory is used for communication between Secure World
& Non Secure World (Structure optee_msg_arg allocated for passing
arguments b/w Secure and Non-Secure World).
> 
> For more information please check Shared Memory section in
OPTEE Design Documentation.
> 
> Some use cases needs this value to be increased.
e.g. In a system where multiple applications are running
simultaneously trying to get services from TA.
> 
> This value needs to be carefully changed based on platform and
use case.

Reply:
Please check this @jbech-linaro @jforissier

Thanks.

Reply:
Please update to as jbech-linaro "would just have written"